File: API/Features/Billing/Invoices/ViewModels/List/InvoiceListVM.cs
using API.Infrastructure.Classes;

namespace API.Features.Billing.Invoices {

    public class InvoiceListVM {

        public string InvoiceId { get; set; }
        public string Date { get; set; }
        public int InvoiceNo { get; set; }
        public SimpleEntity Customer { get; set; }
        public SimpleEntity Destination { get; set; }
        public DocumentTypeVM DocumentType { get; set; }
        public SimpleEntity Ship { get; set; }
        public decimal GrossAmount { get; set; }

    }

}

File: FrontEnd/src/app/features/billing/invoices/classes/view-models/list/invoice-list-vm.ts
import { DocumentTypeVM } from './documentType-vm'
import { InvoiceListAadeVM } from './invoice-list-aade-vm'
import { SimpleEntity } from '../../../../../../shared/classes/simple-entity'

export interface InvoiceListVM {

    customer: SimpleEntity
    date: string
    destination: SimpleEntity
    documentType: DocumentTypeVM
    batch: string
    grossAmount: number
    invoiceId: string
    invoiceNo: number
    formattedDate: string
    isEmailSent: boolean
    ship: SimpleEntity
    shipOwner: SimpleEntity
    aade: InvoiceListAadeVM

}
